### PR TITLE
Mention badger in the main Readme

### PR DESCRIPTION
**Pull request by The-Alchemist**
verbage might need a little tuning

## Which problem is this PR solving?
- 

## Short description of the changes
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ On 28-May-2019, [the OpenTracing and OpenCensus projects announced](https://medi
 
 Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x.
 There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
-with a simple in-memory storage for testing setups.
+with a simple in-memory storage and an [embedded Badger database](https://github.com/dgraph-io/badger) for testing setups.
```
Comment (pavolloffay) [inline README.md]: We are aiming to use badger in prod deployments.

**Comment by pavolloffay** [comment]
@The-Alchemist the commit should be signed with `-s`. See the contributions guidelines.

**Comment by The-Alchemist** [comment]
@pavolloffay 
> @The-Alchemist the commit should be signed with `-s`. See the contributions guidelines.

I'm actually using the Github web UI.

**Comment by pavolloffay** [comment]
You can try adding signed line via the UI.

**Pull request merged**